certified-connectors/snowflake/readme.md

```diff
@@ -25,12 +25,19 @@ create security integration connector
        enabled = true
        external_oauth_type = azure
        external_oauth_issuer = '<AZURE_AD_ISSUER>'     
-       external_oauth_jws_keys_url = 'https://login.windows.net/common/discovery/keys'
-       external_oauth_audience_list = ('https://analysis.usgovcloudapi.net/powerbi/connector/snowflake')
+       external_oauth_jws_keys_url = '<Keys URL from step C>'
```

Comment:
Step 3?

Reply:
Hello @raanandm

May I know the reason that why you reviewed this PR?

Reply:
@vmanoharas  - We have been working with @raanandm to fix some security compliance issues in this connector based on a Feb16th deadline given by Microsoft. @raanandm was trying to expedite the process.

Reply:
Issue has been fixed and committed.